publiccode.yml

```diff
@@ -0,0 +1,64 @@
+# This repository adheres to the publiccode.yml standard by including this 
+# metadata file that makes public software easily discoverable.
+# More info at https://github.com/italia/publiccode.yml
+
+publiccodeYmlVersion: '0.2'
+categories:
+  - it-development
+description:
+  it:
+    documentation: 'https://designers.italia.it/kit/comuni'
+    features:
+      - Template Web
+    genericName: Template siti web Comuni Italiani
+    longDescription: |
+      In questo repository è possibile trovare alcune pagine statiche
+      esemplificative del modello di sito web per i Comuni Italiani. È possibile
+      visualizzare e scaricare il codice HTML/CSS e Javascript dei template, già
+      pronto all’uso e validato in termini di accessibilità e conformità alle
+      Linee guida di design per i servizi web della PA, e costruiti sulle
+      fondamenta fornite dallo UI Kit e dalla libreria Bootstrap Italia.
+
+
+      All'interno del minisito dedicato al modello di sito web per i Comuni
+      Italiani è possibile inoltre trovare alcuni contributi dei comuni che
+      stanno facendo uso del modello di riferimento, tra cui i prototipi del
+      comune di Cagliari. Il repository stesso è un sito in Jekyll e GitHub
+      pages per la condivisione di tali contributi.
+    shortDescription: I template HTML del modello per i siti web dei Comuni Italiani
+developmentStatus: stable
+intendedAudience:
+  countries:
+    - it
+it:
+  conforme:
+    gdpr: true
+    lineeGuidaDesign: true
+    misureMinimeSicurezza: true
+    modelloInteroperabilita: false
+  countryExtensionVersion: '0.2'
+  piattaforme:
+    anpr: false
+    cie: true
+    pagopa: true
+    spid: true
```

Comment:
Uhm, is this right?

Reply:
Not technically, this definition can be easily misunderstood, as such, a simple website can have an area dedicated to technology but not its implementation